std/format.d

```diff
@@ -1120,8 +1121,8 @@ struct FormatSpec(Char)
             {
                 if (trailing.ptr[0] == ' ')
                 {
-                    while (!r.empty && std.ascii.isWhite(r.front)) r.popFront();
-                    //r = std.algorithm.find!(not!(std.ascii.isWhite))(r);
```

Comment:
With selective imports, this does not pose a problem. If `std.utf` suddenly defined `isWhite`, or `std.ascii` defined `stride`, then neither will hijack anything. This is the point of the import improvements.

I only think it's important to worry about conflicts if the current module redefines them.